app/Http/Controllers/User/BookingController.php

```diff
@@ -25,19 +25,19 @@ public function saveClass(Request $request)
     {
         $time = Time::findOrFail($request->time);
         $start = $time->schedule->date . ' ' . $time->time;
-    	$data = [
-    		'student_id' => Auth::id(),
-    		'teacher_id' => $request->teacher,
-    		'start_time' => $start,
-    		'end_time' => Carbon::parse($start)->addHour(2),
-    		'status' => 0,
-    		'link_call' => 'https://meet.google.com/hep-zacz-dhg',
+        $data = [
+            'student_id' => Auth::id(),
+            'teacher_id' => $request->teacher,
+            'start_time' => $start,
+            'end_time' => Carbon::parse($start)->addHour(2),
+            'status' => 0,
+            'link_call' => 'https://meet.google.com/hep-zacz-dhg',
             'name' => Subject::findOrFail($request->subject)->name,
-    	];
+        ];
 
     	$class = Classes::create($data);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Http/Controllers/User/BookingController.php</br>**Line**: 38</br>Spaces must be used to indent lines; tabs are not allowed